pkg/jobrunaggregator/jobrunaggregatorapi/types_row_historical_data.go

```diff
@@ -11,6 +11,8 @@ type HistoricalData interface {
 	GetName() string
 	GetP99() string
 	GetP95() string
+	GetP75() string
+	GetP50() string
```

Comment:
All P50 and P75 are coming out empty string after I ran `hack/run-job-aggregator-analyzer.sh -o ~/go/src/github.com/openshift/origin -g ~/creds/openshift-trt-sippy-consumer-sa.json`

Am I missing something?

Reply:
Updated the query, it was missing the entry